### ISSUE TITLE
Can't locate object method "die" via package "CLI::Osprey::Descriptive::Usage"

### ISSUE THREAD

**Issue by oalders** [arodland/CLI-Osprey#14]
version 0.04

```perl
use strict;
use warnings;

use Test::More;

package Foo;

use Moo;
use CLI::Osprey;

has b => (
    is => 'ro',
);

package main;

local @ARGV = ('-b');
my $foo = Foo->new_with_options;
ok(1);
done_testing();
```

If I run this:

```sh
$ prove -lv t/browse.t
t/browse.t .. Unknown option: b
Can't locate object method "die" via package "CLI::Osprey::Descriptive::Usage" at /Users/olaf/.plenv/versions/5.26.1/lib/perl5/site_perl/5.26.1/Getopt/Long/Descriptive.pm line 441.
```

I'm just trying this module for the first time today.  I haven't dug into the root cause of this.  I'm guessing that my code is just wrong, but the `die` seems wrong too. :)

**Comment by arodland** [comment]
This is fixed by PR #10. I'll cut a release shortly. Thanks for the report.

**Comment by oalders** [comment]
Thanks for your work on this!

**Comment by arodland** [comment]
(Actually there's two issues here ­— the bad "die" is fixed by PR#10, the issue *causing* death is tracked as #5. Fixing #5 is one of the things I hope to do in the next couple days before I ship.
